tools/extract_pyi.py

```diff
@@ -52,11 +52,29 @@ def convert_folder(top_level, stub_directory):
     # Validate that the module is a parseable stub.
     total += 1
     try:
-        astroid.parse(stub_contents)
+        tree = astroid.parse(stub_contents)
+        for i in tree.body:
+            print(i.__dict__['name'])
+            for j in i.body:
+                if isinstance(j, astroid.scoped_nodes.FunctionDef):
+                    a = ''
+                    if None in j.args.__dict__['annotations']:
+                        a += f"Missing parameter type: {j.__dict__['name']} on line {j.__dict__['lineno']}\n"
+                    if j.returns:
+                        if 'Any' in j.returns.__dict__.values():
+                            a += f"Missing return type: {j.__dict__['name']} on line {j.__dict__['lineno']}"
+                    if a:
+                        raise TypeError(a)
```

Comment:
Instead of raising a TypeError and then catching it, please just print out the info directly. Let's not make it fail due to this yet.

Reply:
Ok

Reply:
Made the changes